<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TimeLapseGenMain</class>
 <widget class="QMainWindow" name="TimeLapseGenMain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>996</width>
    <height>665</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="baseSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>TimeLapseGen</string>
  </property>
  <property name="windowIcon">
   <iconset resource="images.qrc">
    <normaloff>:/root/img/Time-Machine-icon.png</normaloff>:/root/img/Time-Machine-icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="topMargin">
     <number>5</number>
    </property>
    <property name="bottomMargin">
     <number>5</number>
    </property>
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>976</width>
         <height>520</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="labelImage">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>996</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionSelect_Images"/>
    <addaction name="action_Enhance_Images"/>
    <addaction name="actionGenerate_Timelapse"/>
    <addaction name="separator"/>
    <addaction name="action_Quit"/>
   </widget>
   <widget class="QMenu" name="menu_Edit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="action_Preferences"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Edit"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="iconSize">
    <size>
     <width>80</width>
     <height>70</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSelect_Images"/>
   <addaction name="separator"/>
   <addaction name="action_Enhance_Images"/>
   <addaction name="separator"/>
   <addaction name="actionGenerate_Timelapse"/>
   <addaction name="separator"/>
   <addaction name="action_Preferences"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionSelect_Images">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/root/img/Camera-icon.png</normaloff>
     <normalon>:/root/img/Camera-icon.png</normalon>:/root/img/Camera-icon.png</iconset>
   </property>
   <property name="text">
    <string>Select  &amp;Images</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="action_Enhance_Images">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/root/img/Pixelmator-icon.png</normaloff>
     <normalon>:/root/img/Television-icon.png</normalon>:/root/img/Pixelmator-icon.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Enhance Images</string>
   </property>
   <property name="toolTip">
    <string>Enhance Images [ Optional ]</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionGenerate_Timelapse">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/root/img/Video-icon.png</normaloff>:/root/img/Video-icon.png</iconset>
   </property>
   <property name="text">
    <string>Generate &amp;Timelapse</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+G</string>
   </property>
  </action>
  <action name="action_Quit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="action_Preferences">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/root/img/Settings-icon.png</normaloff>
     <normalon>:/root/img/Settings-icon.png</normalon>:/root/img/Settings-icon.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Preferences</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
